grakn-engine/src/main/java/ai/grakn/engine/postprocessing/PostProcessingTask.java

```diff
@@ -197,4 +205,26 @@ private boolean runResourceFix(GraknGraph graph, String index, Set<ConceptId> co
     private boolean runCastingFix(GraknGraph graph, String index, Set<ConceptId> conceptIds) {
         return graph.admin().fixDuplicateCastings(index, conceptIds);
     }
+
+    /**
+     * Check if there are duplicate castings for the given index
+     * @param graph Graph on which to apply the fixes
+     * @param index The unique index of the concept that may have duplicates
+     * @param conceptIds The conceptIds which may be the duplicates
+     */
+    private boolean duplicateCastingsExist(GraknGraph graph, String index, Set<ConceptId> conceptIds) {
```

Comment:
Not overly concerned about this one. Just a preference. Feel free to leave as is.